src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -562,7 +562,7 @@ public function getCreated()
      */
     public function setCreator($creator)
     {
-        $this->entity->setChanger($creator);
```

Comment:
this is the setter function for creator in the API object currently it was false set to changer instead to creator